pkgs/applications/networking/remote/aws-workspaces/default.nix

```diff
@@ -0,0 +1,60 @@
+{ stdenv, lib
+, makeWrapper, dpkg, fetchurl, autoPatchelfHook
+, curl, kerberos, lttng-ust, libpulseaudio, gtk3, openssl_1_1, icu, webkitgtk, librsvg, gdk-pixbuf, libsoup, glib-networking
+}:
+
+stdenv.mkDerivation rec {
+  pname = "aws-workspaces";
+  version = "3.1.0.813";
+
+  nativeBuildInputs = [
+    autoPatchelfHook
+    makeWrapper
+  ];
+
+  dontStrip = true;
+
+  buildInputs = [
+    stdenv.cc.cc.lib
+    kerberos
+    curl
+    lttng-ust
+    libpulseaudio
+    gtk3
+    openssl_1_1.out
+    icu
+    webkitgtk
+    librsvg
+    gdk-pixbuf
+    libsoup
+    glib-networking
+  ];
+
+  src = fetchurl {
+    # ref https://d3nt0h4h6pmmc4.cloudfront.net/ubuntu/dists/bionic/main/binary-amd64/Packages
+    url = "https://d3nt0h4h6pmmc4.cloudfront.net/ubuntu/dists/bionic/main/binary-amd64/workspacesclient_${version}_amd64.deb";
+    sha256 = "39ef7117f835fd58e578a4a374f6c2bad18588a36a91d94f3b41c49e947260ff";
+  };
```

Comment:
```suggestion
```

Reply:
Applied in e035973faabfba9299563525986e829ef5753391